netbox/ipam/models/vlans.py

```diff
@@ -253,9 +276,18 @@ def clean(self):
                     ).format(ranges=ranges_to_string(self.group.vid_ranges), group=self.group)
                 })
 
+        # Only Q-in-Q customer VLANs may be assigned to a service VLAN
```

Comment:
Can also set Q-in-Q role without setting any Q-in-Q SVLAN, does that make any sense?